tools.yml

```diff
@@ -1364,6 +1364,8 @@ tools:
     mem: 8
   toolshed.g2.bx.psu.edu/repos/iuc/lumpy_sv/lumpy_sv/.*:
     mem: 8
+  toolshed.g2.bx.psu.edu/repos/iuc/maaslin2/maaslin2/1.16.0+galaxy0:
```

Comment:
Is it only for a specific version or all versions of Maaslin2? You could modify that to a regex like the other tool IDs in this shared DB.

Reply:
no, that is by purpose, only this version cannot handle multiple cores